src/com/cft/contactmerge/contact/tests/NamePropertyTest.java

```diff
@@ -0,0 +1,53 @@
+package com.cft.contactmerge.contact.tests;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import com.cft.contactmerge.contact.*;
+
+
+class NamePropertyTest {
+
+    @Test
+    void NameProperty_Constructor()
+    {
+        NameProperty property = new NameProperty("Doe", "Jane");
+
+        assertNotNull(property);
+    }
+
+    @Test
+    void NameProperty_isMatch() {
+        // TODO: Add tests
+    }
+
+    @Test
+    void NameProperty_getValue() {
+        NameProperty property = new NameProperty("Doe", "Jane");
+
+        assertEquals("Jane", property.getValue().getFirstName(), "Verify FirstName");
+        assertEquals("Doe", property.getValue().getLastName(), "Verify LastName");
```

Comment:
For the message upon failure "Verify *", I would write in a return string method for this (like I did in my CompareContactPartsTest.class (firstAndLastNameFailedMsg). A good rule of thumb is if you hard code something, you should only do it once.

Reply:
Made the change.